Button/Button.module.less

```diff
@@ -305,4 +306,26 @@
 			}
 		});
 	}
+
+	&:global(.spottable):active,
+	:global(.spottable):active & {
+		.applySkins({
+			.bg {
+				background-color: @agate-button-active-bg-color;
+				background-image: none;
```

Comment:
This should be a variable, defined on base as `inherit`, and as `none` in Silicon.
Suggested name: `@agate-button-active-bg-image`